dandischema/__init__.py

```diff
@@ -1,4 +1,3 @@
-__all__ = ["__version__", "migrate", "validate"]
+__all__ = ["__version__"]
 
 from ._version import __version__
-from .metadata import migrate, validate
```

Comment:
This is removed so that importing `dandischema` doesn't import `dandischema.models`.